evm/send-tokens.js

```diff
@@ -31,18 +56,63 @@ async function sendTokens(chain, options) {
         if (anwser !== 'y') return;
     }
 
+    let signersData, transactions;
+    const gasOptions = chain.staticGasOptions || {};
+
+    if (offline) {
+        if (!filePath) {
+            throw new Error('FilePath is not provided in user info');
+        }
+
+        if (nonceOffset) {
+            if (!isValidNumber(nonceOffset)) {
+                throw new Error('Provided nonce offset is not a valid number');
+            }
+
+            nonce += parseInt(nonceOffset);
+        }
+
+        printInfo(`Storing signed Txs offline in file ${filePath}`);
+        signersData = await getAllSignersData(filePath);
+        transactions = await getTransactions(filePath, signerAddress);
```

Comment:
can we skip managing txs via this JSON for now? it'll complicate our flows from before. For e.g. we don't always won't to broadcast all txs from the same signer right away (maybe broadcast for one chain first and test etc.). Instead, I'd just save the signed tx under a [standardized file name](https://github.com/axelarnetwork/key-management-scripts/blob/main/contracts/offline-sign.js#L644) (you can add signer address in the name as well). Maybe if filePath is not passed in, then this approach is used.